options/elf/include/elf.h

```diff
@@ -20,6 +20,7 @@ extern "C" {
 #define SHF_ALLOC 2
 #define SHF_EXECINSTR 4
 #define SHF_STRINGS 16
+#define SHF_INFO_LINK 32
```

Comment:
AFAICT this should be defined to 64, not 32.

Reply:
Oh and SHF_STRINGS should be 32, not 16.

Reply:
And SHF_TLS should be 1024.

Reply:
Addressed as requested.